core/src/main/java/com/sequenceiq/distrox/v1/distrox/service/DistroXService.java

```diff
@@ -66,13 +68,15 @@ private void validate(DistroXV1Request request) {
             throw new BadRequestException(format("If you want to provision a Data Hub then the FreeIPA instance must be running in the '%s' Environment.",
                     environment.getName()));
         }
-        Optional<StackView> datalakeViewByEnvironmentCrn = stackViewService.findDatalakeViewByEnvironmentCrn(environment.getCrn());
-        if (datalakeViewByEnvironmentCrn.isEmpty()) {
-            throw new BadRequestException(format("Data Lake stack cannot be found for environment crn: %s (%s)", environment.getName(), environment.getCrn()));
+        Set<String> sdxCrns = platformAwareSdxConnector.listSdxCrns(environment.getName(), environment.getCrn());
+        if (sdxCrns.isEmpty()) {
+            throw new BadRequestException(format("Data Lake stack cannot be found for environment crn: %s (%s)",
```

Comment:
CRN is uppercase but it might not be needed here as name and CRN is also displayed.